src/battle_transition.c

```diff
@@ -879,7 +879,7 @@ static bool8 BT_Phase2BigPokeball_LoadTilemapAndWave(struct Task *task)
     const u16 *BigPokeballMap = sBigPokeballTilemap;
 
     BT_GetBg0TilemapAndTilesetBase(&tilemapAddr, &tilesetAddr);
-    for (i = 0; i < 20; ++i)
+    for (i = 0; i < 20; i++)
         for (j = 0; j < 30; ++j, ++BigPokeballMap)
```

Comment:
Not important but I really don't see why you have to change var++ into ++var when the expression is not evaluated. And in some cases they were not changed, like this line.

Reply:
To keep consistent between the repos. Most instances of this in pokeemerald has the ++ after the var.

Reply:
I guess it's just confusing because you're doing it in files seemingly unrelated to what you're working on, and as jiang pointed out incompletely. pokeemerald for instance has:
```c
for (i = 0; i < 20; i++)
{
    for (j = 0; j < 30; j++, bigPokeballMap++)
```
Let's maybe keep repo-wide stylistic changes to their own PR.

Reply:
Alright.